contracts/src/LightClient.sol

```diff
@@ -18,7 +18,10 @@ contract LightClient {
     // === Constants ===
     //
     /// @notice System parameter: number of blocks per epoch
-    uint32 public immutable BLOCKS_PER_EPOCH;
+    // TODO make the variable immutable and format it as using  as BLOCKS_PER_EPOCH once we can use
+    // epochs. See
+    // https://github.com/EspressoSystems/espresso-sequencer/issues/940
+    uint64 public blocksPerEpoch;
```

Comment:
why do we need to change this? we still leave this as immutable and set it to `type(uint64).max` in the constructor, right?

also why change type?

Reply:
Yes, I understand now. It is better to set type(uint32).max inside the deployment script.

> also why change type?

Did not realize that $2^{32}$ is a lot of seconds... :)
Fixed in eede4e8.